src/ImageSharp/Processing/Processors/Transforms/Linear/AutoOrientProcessor{TPixel}.cs

```diff
@@ -81,32 +81,32 @@ protected override void OnFrameApply(ImageFrame<TPixel> sourceBase)
         /// Returns the current EXIF orientation
         /// </summary>
         /// <param name="source">The image to auto rotate.</param>
-        /// <returns>The <see cref="OrientationMode"/></returns>
-        private static OrientationMode GetExifOrientation(Image<TPixel> source)
+        /// <returns>The <see cref="ushort"/></returns>
+        private static ushort GetExifOrientation(Image<TPixel> source)
         {
             if (source.Metadata.ExifProfile is null)
             {
-                return OrientationMode.Unknown;
+                return ExifOrientationMode.Unknown;
             }
 
             IExifValue<ushort> value = source.Metadata.ExifProfile.GetValue(ExifTag.Orientation);
             if (value is null)
             {
-                return OrientationMode.Unknown;
+                return ExifOrientationMode.Unknown;
             }
 
-            OrientationMode orientation;
+            ushort orientation;
             if (value.DataType == ExifDataType.Short)
             {
-                orientation = (OrientationMode)value.Value;
+                orientation = value.Value;
             }
             else
             {
-                orientation = (OrientationMode)Convert.ToUInt16(value.Value);
+                orientation = Convert.ToUInt16(value.Value);
                 source.Metadata.ExifProfile.RemoveValue(ExifTag.Orientation);
             }
 
-            source.Metadata.ExifProfile.SetValue(ExifTag.Orientation, (ushort)OrientationMode.TopLeft);
+            source.Metadata.ExifProfile.SetValue(ExifTag.Orientation, ExifOrientationMode.TopLeft);
```

Comment:
Would you be open to make it possible to opt-in to removing the value, instead of resetting it to `ExifOrientationMode.TopLeft`? This would require a single `bool RemoveExifValue { get; }` property and the following code:

```c#
if (this.RemoveExifValue)
{
    source.Metadata.ExifProfile.RemoveValue(ExifTag.Orientation);
}
else
{
    source.Metadata.ExifProfile.SetValue(ExifTag.Orientation, ExifOrientationMode.TopLeft);
}

Reply:
No, we'll keep it since the value was explicitly set by the previous encoder.